nixos/tests/rabbitmq.nix

```diff
@@ -23,5 +50,8 @@ import ./make-test-python.nix ({ pkgs, ... }: {
         'su -s ${pkgs.runtimeShell} rabbitmq -c "rabbitmqctl status"'
     )
     machine.wait_for_open_port(15672)
+    machine.wait_until_succeeds(
+        '${pkgs.rabbitmq-java-client}/bin/PerfTest --time 10 --uri amqp://alice:dJT8isYu6t0Xb6u56rPglSj1vK51SlNVlXfwsRxw@localhost'
```

Comment:
How did you generate `dJT8isYu6t0Xb6u56rPglSj1vK51SlNVlXfwsRxw`? How does it relate to password and config key and password?

Reply:
I guess it's the password? Can you move it to the let block and give it a name?